httpcore5/src/main/java/org/apache/hc/core5/http/protocol/HttpCoreContext.java

```diff
@@ -155,4 +162,10 @@ public HttpResponse getResponse() {
         return getAttribute(HTTP_RESPONSE, HttpResponse.class);
     }
 
+    /**
+     * @since 5.0
+     */
+    public HttpContext getOriginalContext() {
```

Comment:
@ato Do we need to change `HttpCoreContext` at all? This seems like a very special case specific to HTTP/2 push.

Reply:
Good point. I'll remove the getter.

Should the attribute name constant go as well? Do you have a suggestion for a more appropriate class to put the constant in?